newIDE/app/src/InstructionOrExpression/EnumerateInstructions.js

```diff
@@ -63,6 +63,15 @@ const freeInstructionsToRemove = {
   ],
 };
 
+const getExtensionPrefix = (extension: gdPlatformExtension): string => {
```

Comment:
function duplicated? Should be shared maybe?

Reply:
Indeed, I'll import this one from the other file.